### PR TITLE
pep8 indentation, fixes PressLabs/oxygen#1129

### PR DESCRIPTION
**Pull request by aacaz**

### CODE REVIEW

File: silver/admin.py [hunk 1]
```diff
@@ -580,7 +580,7 @@ def perform_action(self, request, queryset, action):
                       'into {results}.'.format(
                           model_name=self._model_name.lower(), count=qs_count,
                           results=results
-                )
+                        )
```
Comment (bogdanpetrea) [inline silver/admin.py]: I think there are 2 too many spaces here.

File: silver/api/urls.py [hunk 2]
```diff
@@ -58,7 +58,8 @@
     url(r'^invoices/(?P<document_pk>[0-9]+)/entries/$',
         views.InvoiceEntryCreate.as_view(), name='invoice-entry-create'),
     url(r'^invoices/(?P<document_pk>[0-9]+)/entries/(?P<entry_pk>[0-9]+)/$',
-        views.InvoiceEntryUpdateDestroy.as_view(), name='invoice-entry-update'),
+        views.InvoiceEntryUpdateDestroy.as_view(
```
Comment (bogdanpetrea) [inline silver/api/urls.py]:
```
views.InvoiceEntryUpdateDestroy.as_view(),
name='invoice-entry-update'),
```
looks better to me.

File: silver/api/urls.py [hunk 3]
```diff
@@ -70,11 +71,13 @@
     url(r'^proformas/(?P<document_pk>[0-9]+)/entries/$',
         views.ProformaEntryCreate.as_view(), name='proforma-entry-create'),
     url(r'^proformas/(?P<document_pk>[0-9]+)/entries/(?P<entry_pk>[0-9]+)/$',
-        views.ProformaEntryUpdateDestroy.as_view(), name='proforma-entry-update'),
+        views.ProformaEntryUpdateDestroy.as_view(
```
Comment (bogdanpetrea) [inline silver/api/urls.py]: Same as above, there is no reason to split a call with no arguments on multiple lines.

File: silver/api/urls.py [hunk 4]
```diff
@@ -70,11 +71,13 @@
...
+        ), name='proforma-entry-update'),
     url(r'^proformas/(?P<pk>[0-9]+)/state/$',
         views.ProformaStateHandler.as_view(), name='proforma-state'),
     url(r'^proformas/(?P<pk>[0-9]+)/invoice/$',
-        views.ProformaInvoiceRetrieveCreate.as_view(), name='proforma-invoice'),
+        views.ProformaInvoiceRetrieveCreate.as_view(
```
Comment (bogdanpetrea) [inline silver/api/urls.py]: Same as above.

File: silver/documents_generator.py [hunk 5]
```diff
@@ -104,9 +104,7 @@ def _generate_for_user_with_consolidated_billing(self, customer,
         for subscription in customer.subscriptions.filter(**criteria):
             if subscription.should_be_billed(billing_date):
                 subs_to_bill.append(subscription)
-                if (force_generate and
-                    subscription.state == Subscription.STATES.ACTIVE
-                ):
+                if (force_generate and subscription.state == Subscription.STATES.ACTIVE):
```
Comment (bogdanpetrea) [inline silver/documents_generator.py]: More than 80 characters on a line.
Maybe
```
                if (force_generate and 
                        subscription.state == Subscription.STATES.ACTIVE):
```

File: silver/documents_generator.py [hunk 6]
```diff
@@ -169,9 +167,7 @@ def _generate_for_user_without_consolidated_billing(self, customer,
...
```
Comment (bogdanpetrea) [inline silver/documents_generator.py]: Same as above.

File: silver/documents_generator.py [hunk 7]
```diff
@@ -169,9 +167,7 @@ def _generate_for_user_without_consolidated_billing(self, customer,
...
                     subscription.cancel(when=Subscription.CANCEL_OPTIONS.NOW)
                     subscription.save()
             elif force_generate and subscription.state == Subscription.STATES.CANCELED:
```
Comment (bogdanpetrea) [inline silver/documents_generator.py]: This line has more than 80 characters. There are a few more in this file.

**Comment by vtemian** [comment]
It's ok to have more than 80 characters, as long as it doesn't screw up the readability (https://www.youtube.com/watch?v=wf-BqAjZb8M)

File: silver/models.py [hunk 8]
```diff
@@ -817,8 +817,7 @@ def add_total_value_to_document(self, billing_date, invoice=None,
 
                     if (self.state == self.STATES.CANCELED and
                         self.cancel_date.month == self.start_date.month and
-                        self.cancel_date == billing_date.month
-                    ):
+                            self.cancel_date == billing_date.month):
```
Comment (bogdanpetrea) [inline silver/models.py]: I'd do 
```
                    if (
                        self.state == self.STATES.CANCELED and
                        self.cancel_date.month == self.start_date.month and
                        self.cancel_date == billing_date.month
                    ):
```
but it works like this too.

File: silver/models.py [hunk 9]
```diff
@@ -974,9 +970,7 @@ def add_total_value_to_document(self, billing_date, invoice=None,
                         if bsd <= self.cancel_date <= bed:
                             bed = self.cancel_date
 
-                    if (self.state == self.STATES.ACTIVE or
-                        self.cancel_date >= self.trial_end
-                    ):
+                    if (self.state == self.STATES.ACTIVE or self.cancel_date >= self.trial_end):
```
Comment (bogdanpetrea) [inline silver/models.py]: More than 80 characters on a line. There are more lines with exceeding characters in this file.

File: silver/tests/commands/test_generate_docs.py [hunk 10]
```diff
@@ -1624,8 +1625,7 @@ def test_gen_for_single_canceled_subscription(self):
                 subscription.cancel(when=Subscription.CANCEL_OPTIONS.NOW)
                 subscription.save()
 
-                call_command('generate_docs', date=billing_date,
-                            subscription=subscription.pk, stdout=self.output)
+                call_command('generate_docs', date=billing_date, subscription=subscription.pk, stdout=self.output)
```
Comment (bogdanpetrea) [inline silver/tests/commands/test_generate_docs.py]: Over 80 characters in one line.

File: silver/tests/commands/test_generate_docs.py [hunk 11]
```diff
@@ -1667,8 +1667,7 @@ def test_gen_active_and_canceled_selection(self):
                     subscription.cancel(when=Subscription.CANCEL_OPTIONS.NOW)
                     subscription.save()
 
-                call_command('generate_docs', billing_date=billing_date,
-                            stdout=self.output)
+                call_command('generate_docs', billing_date=billing_date, stdout=self.output)
```
Comment (bogdanpetrea) [inline silver/tests/commands/test_generate_docs.py]: Over 80 characters.

File: silver/tests/commands/test_generate_docs.py [hunk 12]
```diff
@@ -1678,8 +1677,7 @@ def test_gen_active_and_canceled_selection(self):
 
                 Proforma.objects.all().delete()
 
-                call_command('generate_docs', billing_date=billing_date,
-                            stdout=self.output)
+                call_command('generate_docs', billing_date=billing_date, stdout=self.output)
```
Comment (bogdanpetrea) [inline silver/tests/commands/test_generate_docs.py]: Same as above.

File: silver/tests/commands/test_generate_docs_args.py [hunk 13]
```diff
@@ -65,34 +67,34 @@ def setUp(self):
...
+
         assert self.output.getvalue() == self.good_output
 
     def test_generate_docs_subscription_argparser(self):
 
-        call_command('generate_docs', '--subscription=%s' % self.subscription.id,
-                      stdout=self.output)
+        call_command(
+            'generate_docs', '--subscription=%s' % self.subscription.id,
+            stdout=self.output)
```
Comment (bogdanpetrea) [inline silver/tests/commands/test_generate_docs_args.py]:
```
stdout=self.output
)
```

File: silver/tests/factories.py [hunk 14]
```diff
@@ -128,9 +128,8 @@ class Meta:
     customer = factory.SubFactory(CustomerFactory)
     start_date = timezone.now().date()
     trial_end = factory.LazyAttribute(
-        lambda obj: obj.start_date +\
-                        datetime.timedelta(days=obj.plan.trial_period_days)
-                    if obj.plan.trial_period_days else None)
+        lambda obj: obj.start_date + datetime.timedelta(days=obj.plan.trial_period_days)
```
Comment (bogdanpetrea) [inline silver/tests/factories.py]: More than 80 characters.

File: silver/tests/spec/test_customer.py [hunk 15]
```diff
@@ -72,8 +72,8 @@ def test_create_post_customer_without_required_field(self):
                                         content_type='application/json')
 
             self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-            assert (response.data == {field: ['This field may not be blank.']}
-                    or response.data == {field: ['This field is required.']})
+            assert (response.data == {field: ['This field may not be blank.']} or
```
Comment (bogdanpetrea) [inline silver/tests/spec/test_customer.py]: Over 80 characters.

File: silver/tests/spec/test_invoice.py [hunk 16]
```diff
@@ -468,9 +472,10 @@ def test_issue_invoice_when_in_issued_state(self):
         url = reverse('invoice-state', kwargs={'pk': 1})
         data = {'state': 'issued'}
         response = self.client.put(url, data=json.dumps(data),
-                                     content_type='application/json')
+                                   content_type='application/json')
         assert response.status_code == status.HTTP_403_FORBIDDEN
-        assert response.data == {'detail': 'An invoice can be issued only if it is in draft state.'}
+        assert response.data == {
+            'detail': 'An invoice can be issued only if it is in draft state.'}
```
Comment (bogdanpetrea) [inline silver/tests/spec/test_invoice.py]:
```
'detail': 'An invoice can be issued only if it is in draft state.'
}
```

File: silver/tests/spec/test_invoice.py [hunk 17]
```diff
@@ -483,9 +488,10 @@ def test_issue_invoice_when_in_paid_state(self):
...
```
Comment (bogdanpetrea) [inline silver/tests/spec/test_invoice.py]:
```
'detail': 'An invoice can be issued only if it is in draft state.'
}
```

File: silver/tests/spec/test_invoice.py [hunk 18]
```diff
@@ -547,9 +552,10 @@ def test_pay_invoice_when_in_draft_state(self):
         url = reverse('invoice-state', kwargs={'pk': 1})
         data = {'state': 'paid'}
         response = self.client.put(url, data=json.dumps(data),
-                                     content_type='application/json')
+                                   content_type='application/json')
         assert response.status_code == status.HTTP_403_FORBIDDEN
-        assert response.data == {'detail': 'An invoice can be paid only if it is in issued state.'}
+        assert response.data == {
+            'detail': 'An invoice can be paid only if it is in issued state.'}
```
Comment (bogdanpetrea) [inline silver/tests/spec/test_invoice.py]:
```
            'detail': 'An invoice can be paid only if it is in issued state.'
        }
```

File: silver/tests/spec/test_metered_feature.py [hunk 19]
```diff
@@ -70,8 +71,8 @@ def test_create_post_metered_feature_without_required_field(self):
                                         content_type='application/json')
 
             self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-            assert (response.data == {field: ['This field may not be blank.']}
-                    or response.data == {field: ['This field is required.']})
+            assert (response.data == {field: ['This field may not be blank.']} or
```
Comment (bogdanpetrea) [inline silver/tests/spec/test_metered_feature.py]: More than 80 characters on line.

**Comment by bogdanpetrea** [comment]
There are many lines exceeding 80 characters. Those should be fixed too.

**Pull request merged**